### ISSUE TITLE
Issues with guestbook sample - segfault ?

### ISSUE THREAD

**Issue by robmaw** [radius-project/samples#1005]
Running radius 0.31 with minikube on docker ce on WSL.

I'm trying to follow the tutorial "https://docs.radapp.io/tutorials/add-radius/" but am running into an issue where the frontend and redis-replica deployments do not succeed - i.e. after using kubectl apply .... to perform the deployment, all the frontend and replica pods crash and restart without any logs.
describe pod returns some info:
```
>kubectl describe pod redis-replica -n demo
Name:             redis-replica-86f8895958-6hff5
Namespace:        demo
... snip ...
    Port:           6379/TCP
    Host Port:      0/TCP
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    139
...snip ....
```
the main redis deployment works, and the getting started tutorial works fine too.

[AB#11417](https://dev.azure.com/azure-octo/e61041b4-555f-47ae-95b2-4f8ab480ea57/_workitems/edit/11417)

**Comment by willtsai** [comment]
@willdavsmith - I think this might be related to the issue you were fixing related to upmerges of branches within the `samples` repo. I think a downmerge from `edge` to `v0.32` now should fix this? I recommend trying the sample on the `edge` branch and if it's not reproducible there, then a downmerge should address this issue.

**Comment by willtsai** [comment]
@robmaw -- can you please try this again on the latest Radius build and branch (`v0.33`)? This issue should now be resolved, let me know if it works for you.

**Comment by willtsai** [comment]
since this should have been resolved with release `v0.33` via https://github.com/radius-project/samples/pull/1212, I'll close this for now. please reopen if any issues.
